### PR TITLE
Mobile Menu for Portfolio project

### PR DESCRIPTION
**Pull request by SurafelGetachewT**
In this project, we collaborate to:
1. Add the mobile menu for the portfolio project.
2. Use javascript to manipulate DOM elements of mobile menu
3. Use javascript events on mobile menu.

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -824,6 +824,36 @@ form button:hover {
...
+  height: 100%;
+  width: 100%;
+  left: 0;
+  right: 0;
+  top: 0;
+  background-color: #6070fff0;
+  mix-blend-mode: multiply;
+  text-align: left;
+  display: none;
+}
```
Comment (zieeco) [inline css/style.css:838]: - Please, kindly ensure your design is close to the Figma design as much as you can below:
   - Adjust the button size and color
   - Align the list items correctly.
 

 *figma design*
mooo

*your design*
mo

Comment (zieeco) [reply css/style.css:838]: *README FILE*

- Please, kindly add a getting started header section and add some instructions on how to have your project up and running locally on users' machines.
- Add a license and the license contained in it as the MIT.md file is a broken link to make your readme more professional
- Add the acknowledgment and its content with appropriate credit to Figma for using their design template.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @SurafelGetachewT ,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by vickymarz — APPROVED** [review]
Hi @SurafelGetachewT , 

# 🟢  Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**